src/wp-admin/includes/noop.php

```diff
@@ -104,3 +104,19 @@ function get_file( $path ) {
 
 	return @file_get_contents( $path );
 }
+
+/**
+ * @ignore
+ * @since 6.2.0
+ */
+function wp_cache_get() {
+	return false;
+}
```

Comment:
```suggestion
function wp_cache_get( $key = '', $group = '', $force = false, &$found = null ) {
	$found = false;
	return false;
}
```

This is to account for the case in which `$found` is used to account for `false` as a possible cached value.

https://github.com/WordPress/wordpress-develop/blob/5a497225b286964ff9302a21272731172840ed05/src/wp-includes/class-wp-query.php#L3128-L3130

I don't know how much it's used in core but it would be good to avoid an error if it is.